routes/laughboxes.js

```diff
@@ -16,27 +16,27 @@ const laughboxNotFoundError = (id) => {
 }
 
 // Laughbox landing page
-router.get('/', csrfProtection, (req, res) => {
-  // const laughbox = db.Laughbox.build();
-  const errors = [];
-
-  if (!req.session.user) {
-    errors.push("You must log in to view your laughbox!")
-    res.render('laughboxes-logged-out', {
-      title: 'Please log in',
-      errors,
-    })
-  }
```

Comment:
I think it's a nice feature to prevent non-login users to access laughboxes since it's a uer-only feature. What do you think?

Reply:
I agree, that was my bad. I didn't mean to take this out. Good catch!